pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -403,7 +403,11 @@ def select_precision_plugin(self) -> PrecisionPlugin:
         raise NotImplementedError("We only support precisions 64, 32 and 16!")
 
     def select_training_type_plugin(self) -> TrainingTypePlugin:
-        if self.use_ddp2:
+        if isinstance(
+            self.distributed_backend, Accelerator
+        ) and self.distributed_backend.training_type_plugin is not None:
+            plugin = self.distributed_backend.training_type_plugin
```

Comment:
Nice !

Reply:
Mind adding a test ?

Reply:
done.

Reply:
do we need to add the same fix to the `select_precision_plugin()` method?